dataland-e2etests/src/test/kotlin/org/dataland/e2etests/tests/DataOwnerControllerTest.kt

```diff
@@ -158,7 +158,7 @@ class DataOwnerControllerTest {
         }
         assertErrorCodeForClientException(headExceptionForNotFoundDataOwner, 404)
         jwtHelper.authenticateApiCallsWithJwtForTechnicalUser(
-            TechnicalUser.entries.filter { it != TechnicalUser.Admin }.random(),
+            TechnicalUser.values().filter { it != TechnicalUser.Admin }.random(),
```

Comment:
Why this change?

Reply:
It seems to be a version problem. While my IDE suggests .entries(), Pauls IDE only works with .values() at this point. .entries should be the bettwer solution.

Reply:
ok then i leave it open to change it back to entries if there is no real reason to switch to values

Reply:
Yes, this was due to a deprecated IDE version. Should be reverted now, such that the conversion can be closed

Reply:
ok